practice-app/website/test/post_comment_test.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+import requests, json
+from ..api.event import check_comment_has_text
+
+class TestEventAPI(unittest.TestCase):
+
+    # given an empty comment, it returns false
+    def test_comment_has_test(self):
```

Comment:
One test with the correct output can be created

Reply:
Thank you, added.